pkg/syncer/specsyncer.go

```diff
@@ -155,23 +155,19 @@ func (c *Controller) applyToDownstream(ctx context.Context, gvr schema.GroupVers
 	downstreamObj.SetNamespace(downstreamNamespace)
 	downstreamObj.SetManagedFields(nil)
 	downstreamObj.SetClusterName("")
-
-	ownedByLabel := downstreamObj.GetLabels()["kcp.dev/owned-by"]
```

Comment:
Note to other reviewers: OwnerReferences being keyed to UID means they aren't portable across clusters.